libPatches.py

```diff
@@ -0,0 +1,2533 @@
+def get_lib_patches():
+    return {
+        "osx-arm64": [
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "f378f889797edd7df8d32272c06ca80a1b6b0f58",
+                        "size": 13164,
+                        "url": "https://libraries.minecraft.net/com/mojang/text2speech/1.11.3/text2speech-1.11.3.jar",
+                    }
+                },
+                "name": "com.mojang:text2speech:1.11.3",
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "369a83621e3c65496348491e533cb97fe5f2f37d",
+                        "size": 91947,
+                        "url": "https://github.com/MinecraftMachina/Java-Objective-C-Bridge/releases/download/1.1.0-mmachina.1/java-objc-bridge-1.1.jar",
+                    }
+                },
+                "name": "ca.weblite:java-objc-bridge:1.1.0",
+            },
+            {
+                "downloads": {
+                    "classifiers": {
+                        "natives-osx": {
+                            "sha1": "53f9c919f34d2ca9de8c51fc4e1e8282029a9232",
+                            "size": 12186,
+                            "url": "https://libraries.minecraft.net/net/java/jinput/jinput-platform/2.0.5/jinput-platform-2.0.5-natives-osx.jar",
+                        }
+                    }
+                },
+                "extract": {"exclude": ["META-INF/"]},
+                "name": "net.java.jinput:jinput-platform:2.0.5",
+                "natives": {"osx": "natives-osx"},
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "39c7796b469a600f72380316f6b1f11db6c2c7c4",
+                        "size": 208338,
+                        "url": "https://libraries.minecraft.net/net/java/jinput/jinput/2.0.5/jinput-2.0.5.jar",
+                    }
+                },
+                "name": "net.java.jinput:jinput:2.0.5",
+            },
+            {
+                "downloads": {
+                    "artifact": {
+                        "sha1": "e12fe1fda814bd348c1579329c86943d2cd3c6a6",
+                        "size": 7508,
+                        "url": "https://libraries.minecraft.net/net/java/jutils/jutils/1.0.0/jutils-1.0.0.jar",
+                    }
+                },
+                "name": "net.java.jutils:jutils:1.0.0",
+            },
```

Comment:
What's up with these? `jinput` and `jutils` both seem to be 2.0.5 and 1.0.0 respectively anyway for all versions of the game.

And jinput-platform just uses the existing natives for arm64? That won't really work I assume

Reply:
We need to include the jinput and jutils jars in libPatches or else they'll get disabled for macos and linux arm.
No one really built jinput-platform on arm so I just left them as the regular ones.

Building jinput now